src/normalizer.ts

```diff
@@ -222,6 +223,29 @@ rules.set('Transform const to singleton enum', schema => {
   }
 })
 
+rules.set('Transform nullable to null type', schema => {
+  if (schema.nullable !== true) {
+    return
+  }
+
+  delete schema.nullable
+
+  // schema.const has already been converted to a single-value enum by rule "Transform const to singleton enum", so it
+  // does not need to be handled here.
```

Comment:
Should `schema.const` handling be added in this normalizer in case rules are reordered in the future and "Transform const to singleton enum" no longer precedes this one?